app/views/savedForLater.scala.html

```diff
@@ -16,82 +16,66 @@
 
 @import models.pages.Summary
 @import org.joda.time.LocalDate
-@import config.ForConfig
-
-@(summary: Summary, password: String, expiryDate: LocalDate, hasTimedout: Boolean = false)(implicit requestHeader: RequestHeader, messages: Messages)
-
-@headExtra = {
-@views.html.includes.stylesheets()
-@views.html.includes.analytics(ForConfig.gtm, ForConfig.analytics.ga.startDate, ForConfig.analytics.ga.returnCount, ForConfig.analytics.ga.refNum, summary)
-}
-
-@bodyEnd = {
-@views.html.includes.javascripts()
+@import uk.gov.hmrc.govukfrontend.views.html.components.{GovukButton, GovukWarningText,GovukPanel}
+@import uk.gov.hmrc.govukfrontend.views.Aliases.HtmlContent
+@import uk.gov.hmrc.govukfrontend.views.Aliases.Content
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.warningtext.WarningText
+
+@this(main_template: main2, govukWarningText: GovukWarningText, govukButton: GovukButton, govukPanel: GovukPanel)
+
+@(summary: Summary, password: String, expiryDate: LocalDate, hasTimedOut: Boolean = false)(implicit request: Request[_],  messages: Messages)
+
+@import uk.gov.hmrc.govukfrontend.views.Aliases.Panel
```

Comment:
Move import to the where are other imports. IntelliJ have problem to parse this.